src/test/java/com/syncapse/jenkinsci/plugins/awscloudformationwrapper/ParametersParserTest.java

```diff
@@ -0,0 +1,40 @@
+package com.syncapse.jenkinsci.plugins.awscloudformationwrapper;
```

Comment:
Very, very nice to see unit tests! If you move the methods as mentioned above you can also unit test merging as well.